pygments/lexers/configs.py

```diff
@@ -729,6 +729,13 @@ def heredoc_callback(self, match, ctx):
                 heredoc_callback,
             )
         ],
+        'blockname': [
+            # e.g. resource "aws_security_group" "allow_tls" {
+            # e.g. backend "consul" {
+            (r'(\s*)("[0-9a-zA-Z-_]+")?(\s*)("[0-9a-zA-Z-_]+")',
+             bygroups(Whitespace, Name.Class, Whitespace, Name.Variable, Whitespace, Punctuation)),
```

Comment:
Fails with `regexlint` in the CI, it complains that there are only 4 groups for 6 token types in bygroups(). I tend to agree :-)